srcpkgs/brother-hl5340d-cupswrapper/files/LICENSE

```diff
@@ -0,0 +1,9 @@
+License Agreement
```

Comment:
Licenses should not be vendored

Reply:
[Should fetch license here](https://support.brother.com//g/s/agreement/English_lpr/agree.html) and then vlicense it.

Reply:
Also the package should be marked `nonfree=yes`